contracts/Vault.vy

```diff
@@ -888,19 +908,9 @@ def _shareValue(shares: uint256) -> uint256:
     # Determines the current value of `shares`.
         # NOTE: if sqrt(Vault.totalAssets()) >>> 1e39, this could potentially revert
     lockedFundsRatio: uint256 = (block.timestamp - self.lastReport) * self.lockedProfitDegration
```

Comment:
This one can be removed.

Reply:
4165d877af771a7918d98630d7031705a74c1142